sink-connector/src/test/kotlin/org/neo4j/connectors/kafka/sink/Neo4jSinkRawJSONIT.kt

```diff
@@ -0,0 +1,168 @@
+ * limitations under the License.
+ */
+package org.neo4j.connectors.kafka.sink
+
+import io.kotest.matchers.shouldBe
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.toJavaDuration
+import org.apache.kafka.connect.data.Schema
+import org.apache.kafka.connect.data.SchemaBuilder
+import org.awaitility.Awaitility.await
+import org.junit.jupiter.api.Test
+import org.neo4j.connectors.kafka.testing.format.KafkaConverter.JSON_RAW
+import org.neo4j.connectors.kafka.testing.format.KeyValueConverter
+import org.neo4j.connectors.kafka.testing.kafka.ConvertingKafkaProducer
+import org.neo4j.connectors.kafka.testing.sink.CypherStrategy
+import org.neo4j.connectors.kafka.testing.sink.Neo4jSink
+import org.neo4j.connectors.kafka.testing.sink.TopicProducer
+import org.neo4j.driver.Session
+
+@KeyValueConverter(key = JSON_RAW, value = JSON_RAW)
+class Neo4jSinkRawJSONIT {
+  companion object {
+    private const val TOPIC = "persons"
+  }
+
+  @Neo4jSink(
+      cypher =
+          [
+              CypherStrategy(
+                  TOPIC,
+                  "WITH __value AS person MERGE (p:Person {name: person.name, surname: person.surname})",
+              ),
+          ],
+  )
+  @Test
+  fun `should support json map`(
+      @TopicProducer(TOPIC) producer: ConvertingKafkaProducer,
+      session: Session,
+  ) {
+    producer.publish(
+        value = mapOf("name" to "Jane", "surname" to "Doe"),
+        valueSchema = SchemaBuilder.map(Schema.STRING_SCHEMA, Schema.STRING_SCHEMA).build(),
+    )
+
+    await().atMost(30.seconds.toJavaDuration()).until {
+      session
+          .run(
+              "MATCH (p:Person {name: \$name, surname: \$surname}) RETURN count(p) = 1 AS result",
+              mapOf("name" to "Jane", "surname" to "Doe"),
+          )
+          .single()["result"]
+          .asBoolean()
+    }
+  }
+
+  @Neo4jSink(
+      cypher =
+          [
+              CypherStrategy(
+                  TOPIC,
+                  "WITH __value AS persons UNWIND persons AS person MERGE (p:Person {name: person.name, surname: person.surname})",
+              ),
+          ],
+  )
+  @Test
+  fun `should support json list`(
+      @TopicProducer(TOPIC) producer: ConvertingKafkaProducer,
+      session: Session,
+  ) {
+    producer.publish(
+        value =
+            listOf(
+                mapOf("name" to "Jane", "surname" to "Doe"),
+                mapOf("name" to "John", "surname" to "Doe"),
+            ),
+        valueSchema =
+            SchemaBuilder.array(
+                    SchemaBuilder.map(
+                            Schema.STRING_SCHEMA,
+                            Schema.STRING_SCHEMA,
+                        )
+                        .build(),
+                )
+                .build(),
+    )
+
+    await().atMost(30.seconds.toJavaDuration()).untilAsserted {
+      session
+          .run(
+              "MATCH (p:Person) RETURN count(p) as result",
```

Comment:
```suggestion
              "MATCH (p:Person) WHERE [p.name, p.surname] IN $names  RETURN count(p) as result",
```
`$names` would be `[['Jane', 'Doe'], ['John', 'Doe']]`